fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/VarArgFunction.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+/**
+ * A function with a variable number of arguments
+ */
+public abstract class VarArgFunction extends FunctionExpression {
```

Comment:
Intent of this class? It's only extended by Coalesce and not called. 

I think it deserves an @implnote  indicating the intent

Reply:
removed